Source/Request/Request.JSONP.js

```diff
@@ -119,19 +119,20 @@ Request.JSONP = new Class({
 		return !!this.running;
 	},
 
-	clear: function(){
+	clear: function(index){
```

Comment:
i considered it but it's so short lived - initially even thought of storage but that meant storage gc when element is removed. i'd agree about `data-prop` as it ought to be slightly cheaper.